### ISSUE TITLE
Is it possible to build and run RhinoLink on a Mac system?

### ISSUE THREAD

**Issue by joohaeng** [WolframResearch/RhinoLink#2]
Is it possible to build and run RhinoLink on a Mac system?

Thank you for the wonderful project!

/JH

**Comment by arnoudbuzing** [comment]
I think the answer is "No", but I have assigned the person who would know for sure.  One of the dependencies is NETLink, which is a Windows software component. See also: http://blog.rhino3d.com/2018/09/rhinolink-for-wolfram-language.html

Thank you for your interest.

**Comment by carlsongithub** [comment]
No, RhinoLink is not yet supported on Mac. It's the next item on our agenda, but I can't tell you when to expect it.
